### DISCUSSION TITLE
Canvas animations in Docusaurus markdown possible?

### DISCUSSION THREAD

**Discussion by kayx23** [facebook/docusaurus#8897] [Q&A]
Hello! Is it possible to add canvas animation into markdown docs *somehow*?

**Comment by slorber** [comment]
I have no idea what you mean here 😅 

Please provide a proper explanation of what you mean by canvas animation, and eventually external links

**Reply by kayx23** [reply]
Embedding animations created using Canvas html tag, CSS, and JS, into markdown files. But I've just noticed Docusaurus supports embedding React(?) components into markdown files. I think that could be utilized. Closing this discussion now :)
